README.md

```diff
@@ -5,18 +5,29 @@
 ## Overview
 
 This repo provides a proof-of-concept implementation of the Truebit smart contract system.<br/>
+
 It enables a step-by-step running and verification of the [Scrypt](https://en.wikipedia.org/wiki/Scrypt) hashing function on the EVM.<br/>
+
 This can form the basis of a Dogecoin/Ethereum relay. Dogecoin uses Scrypt for its Proof-of-Work algorithm, specifically to calculate the block hash. Therefore, an ability to compute Scrypt hashes is required in order to verify the validity of Dogecoin transactions, as it allows one to check Merkle proofs from a known block header. This computation is too large given Ethereum's gas constraints, making it an ideal use case for Truebit.<br/>
+
 For more context on how relays work, look at the [BTCRelay](https://github.com/ethereum/btcrelay) project.<br/>
 
-## Running the Tests
+![Protocol Schematic](./doge-eth.png)
 
-To run the tests you need `geth` and a JS `node` version higher than 7.6. After getting those you can just run `run.sh` in the root directory of the repo.<br/>
-Also note that you will need to have an account in the data directory that geth has been passed. `run.sh` does not make one for you. You can either copy an existing account there or just run:<br/>
-```bash
+## Running the Tests
 
-geth --datadir /tmp/ethereum_dev_mode --password /path/to/password/file account new
+After cloning the repo, make sure you have the zeppelin libraries. If you want to use `run.sh`, also make sure you have `ganache-cli`:<br/>
+```
+#To get the zeppelin solidity libraries:
+npm install zeppelin-solidity
 
+#To geth ganache-cli(formerly testrpc):
+npm install ganache-cli
 ```
+After that just run `./run.sh`.<br/>
+
+If you want to run anything other than `ganache-cli`, run it and then run `truffle test`.<br/>
+
+## Tool Versions
 
-Also make sure that when you run a process in the background it is not suspended. You could use `ps -aux | grep geth` to see geth's status. if it is suspended, just run `stty tostop` to toggle your terminal's behavour.<br/>
+For solidity, make sure you are using `0.18.0` or up. Refrain from using nightly builds. If you have to build your solidity, first look at [here](https://github.com/ethereum/solidity/releases) and then get the source from the tarball(don't clone the solidity repo).<br/>
```

Comment:
You probably mean 0.4.18

Reply:
Updated the README in https://github.com/sinahab/scrypt-interactive/commit/6d664f1638a2ed2bd5e24b9e4b17aefba10dfc27.